jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/HttpFieldsTest.java

```diff
@@ -371,9 +385,19 @@ public void testCaseInsensitive()
         assertThat(header.get("Foo-Bar"), is("one"));
         assertThat(header.getValuesList("foo-bar"), contains("one", "two"));
         assertThat(header.getValuesList("Foo-Bar"), contains("one", "two"));
+        assertTrue(header.contains("foo-bar"));
+        assertTrue(header.contains("Foo-Bar"));
 
         // We know the order of the set is deterministic
-        assertThat(header.getFieldNamesCollection(), contains("expect", "RaNdOm", "Accept-Charset", "foo-bar"));
+        Set<String> names = header.getFieldNamesCollection();
```

Comment:
You should test `iterator()` too.

Reply:
I'm pretty sure that is what `contains` is doing?  Or do you mean the header iterator?